readthedocs.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.8"
+    nodejs: "14"
```

Comment:
Do we also need `yarn`?

Reply:
No, I use npm to build the TypeDoc since it comes with nodejs, it is easier to setup.

Reply:
When I built the dos locally, it failed because `yarn` was not installed. Maybe you changed it in recent commits?

Reply:
Fixed in #96.